src/components/Gameover/Gameover.js

```diff
@@ -9,12 +9,12 @@ const Gameover = ({ handleClick, isVisible }) => {
       }
     >
       <p className={styles.header}>
-        Gratulacje,
+        Congratulations,
         <br />
-        wygrałaś!
+        you won!
       </p>
       <button className={styles.button} onClick={handleClick}>
-        Rozpocznij od nowa
```

Comment:
Here, since the button looks almost the same as menu buttons think about reuse of the button component with different styling (different background as far as I see). You can use `children` prop.

Reply:
Sure, I'm going to fix that.